docs/recipes/IsolatingSubapps.md

```diff
@@ -0,0 +1,81 @@
+# Isolating Redux Subapps
+
+Consider the case of a “big” app (contained in a `<BigApp>` component)
+that embeds smaller “subapps” (contained in `<SubApp>` components):
```

Comment:
Nit: "subapp" isn't a word. Can you change any instances to "sub-app", where appropriate?